nicegui/js.py

```diff
@@ -0,0 +1,18 @@
+from pathlib import Path
+from typing import Optional
+
+from .nicegui import app
+
+
+class js(str):
+
+    def __new__(cls, obj: str, file: Optional[Path] = None):
+        val: str
+        if file:
+            file_resolved = file.resolve()
+            assert file_resolved.is_file() and file_resolved.suffix == '.js', f'{file_resolved} is not a .js file.'
+            app.add_static_file(local_file=file_resolved, url_path=f'/user-static-js/{file.stem}')
+            val = f'_nicegui_js_import:{{"url": "/user-static-js/{file.stem}", "obj": "{obj}"}}'
+        else:
+            val = f'_nicegui_need_to_eval_:{obj}'
```

Comment:
@jdoiro3 - think about these special string 'keys' more. Should they be something else?